src/compose/app.ts

```diff
@@ -336,15 +335,10 @@ export class App {
 				return false;
 			}
 
-			// Check if we previously remember starting it
-			if (
-				applicationManager.containerStarted[serviceCurrent.containerId!] != null
-			) {
-				return false;
-			}
-
-			// If the config otherwise matches, then we should be running
-			return isEqualExceptForRunningState(serviceCurrent, serviceTarget);
+			// Only start a Service if we have never started it before!
+			// Otherwise, all other states regarding starting will be handled by the engine.
```

Comment:
What is meant by "all other states regarding starting"?

Reply:
I mean that there are N possible status states and the only time we want to start a container is when it is Installed. Otherwise, all other states will be determined by the engine if it should be started.

I agree it's not clear >.<

Reply:
updated